resources/language/English/strings.po

```diff
@@ -180,3 +180,7 @@ msgstr ""
 msgctxt "#30048"
 msgid "WEEK {0}"
 msgstr ""
+
+msgctxt "#30049"
+msgid "Use InputStream Adaptive"
```

Comment:
I think this is a bit technical for users. I'd just call it "Use Adaptive Bitrate" or "Use Adaptive Streaming". Something that doesn't use the Kodi internals name. ;-)